app/experimenter/nimbus-ui/src/components/FormBranches/index.tsx

```diff
@@ -0,0 +1,228 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import Form from "react-bootstrap/Form";
+import Col from "react-bootstrap/Col";
+import Button from "react-bootstrap/Button";
+import Badge from "react-bootstrap/Badge";
+import { ReactComponent as DeleteIcon } from "../../images/x.svg";
+
+import {
+  getExperiment_experimentBySlug,
+  getExperiment_experimentBySlug_treatmentBranches,
+} from "../../types/getExperiment";
+import { getConfig_nimbusConfig } from "../../types/getConfig";
+
+/* TODO: EXP-505 for managing branch editing interactions & state */
+
+const FormBranches = ({
+  experiment,
+  equalRatio,
+  featureConfig,
+}: {
+  experiment: getExperiment_experimentBySlug;
+  equalRatio?: boolean;
+  featureConfig: getConfig_nimbusConfig["featureConfig"];
+}) => {
+  const {
+    referenceBranch,
+    treatmentBranches,
+    featureConfig: experimentFeatureConfig,
+  } = experiment!;
+  return (
+    <section data-testid="FormBranches" className="border-top my-3">
+      <Form className="p-2">
+        <Form.Row className="my-3">
+          <Form.Group controlId="evenRatio">
+            <Form.Check
+              type="checkbox"
+              label="Users should be split evenly between all branches"
+            />
+          </Form.Group>
+          <Form.Group as={Col} className="align-top text-right">
+            <Button variant="outline-primary" size="sm">
+              + Add branch
+            </Button>
+          </Form.Group>
+        </Form.Row>
+      </Form>
+      <section>
+        {referenceBranch && (
```

Comment:
I assume "New" experiments should start with an empty reference branch, yeah? Maybe for the next issue?
<img width="885" alt="Screen Shot 2020-10-30 at 7 25 59 PM" src="https://user-images.githubusercontent.com/6392049/97764713-e6fb6480-1ae5-11eb-835e-ce93285375c4.png">

Reply:
Not doing anything with the data until EXP-505 / issue #3781